src/frontend/static/icons/Cymbal_NavLogo.svg

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I've used the name `Cymbal_NavLogo.svg` here because the "Online Boutique" logo/SVG file is named `Hipster_NavLogo.svg`.